src/node.cpp

```diff
@@ -87,8 +87,8 @@ Node::Node(MegaClient* cclient, node_vector* dp, handle h, handle ph,
 
     plink = NULL;
 
-    memset(&changed,-1,sizeof changed);
-    changed.removed = false;
+    memset(&changed, 0, sizeof changed);
+    changed.newnode = true;
```

Comment:
It might be clearer to keep it false in constructor and set the flag at `MegaClient::readnodes` call to new Node.

Reply:
I like the idea. I'll change it. Thanks